docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -123,3 +123,87 @@ Use *\\* for escaping of special characters in regular expressions
 * To consider failed assertion known or potentially-known *assertionPattern* should match assertion failure, if defined *variablePatterns* should match also.
 * If *assertionPattern*/*variablePatterns* matched and you have *story/scenario/step/Pattern* defined, to consider issue known all of them should match.
 * If *assertionPattern*/*variablePatterns* matched and you have *story/scenario/step/Pattern* defined, when at least one not matched issue is considered potentially known.
+
+== Meta filters
+
+Meta tags functionality can be used to reach several goals:
+
+* Scenarios' grouping in Allure report.
+* Scenarios' delimitation for test executing by selected tag.
+* Setting desired execution attributes on scenario-level.
+
+=== Levels
+Meta tags may be used at both story and scenario level. Meta tags specified at Story level are propagated to every scenario, if the same meta tag is used at Scenario level, then Scenario level value is applied.
+----
```

Comment:
please format examples in the same way they formatted across other pages,
 e.g.: https://raw.githubusercontent.com/vividus-framework/vividus/master/docs/modules/commons/pages/vividus-steps.adoc -> Example ".Click button 5 times" ->
```
.Click button 5 times
[source,gherkin]
----
When I execute steps at most 5 times while variable `var` is less than `3`:
|step                                                                                                                   |
|When I click on element located `id(counter)`                                                                          |
|When I find <= `1` elements by `xpath(//div[@id='clickResult' and (text()='3' or text()='4')])` and for each element do|
|{headerSeparator=!,valueSeparator=!}                                                                                   |
|!step!                                                                                                                 |
|!When I set the text found in search context to the 'scenario' variable 'var'!                                         |
Then `${var}` is = `3`
----
```

Reply:
the same for all other examples